internal/engine/wazevo/backend/backend_test.go

```diff
@@ -1713,6 +1713,63 @@ L1 (SSA Block: blk0):
 	fcvt d1, s1
 	ldr x30, [sp], #0x10
 	ret
+`,
+		},
+		{
+			name: "nontrapping_float_conversions",
+			m:    testcases.NonTrappingFloatConversions.Module,
+			afterLoweringARM64: `
```

Comment:
nothing interesting here, so let's not have `afterLoweringARM64` and only have `afterFinalizeARM64` (the assertion on `afterLoweringARM64` will be skipped if it is empty).

Reply:
@evacchi I believe this hasn't be resolved.

Reply:
maybe I reset it by mistake